### ISSUE TITLE
`geom_bar` does not support negative values

### ISSUE THREAD

**Issue by dizzyliam** [Vindaar/ggplotnim#64]
### Issue:
`geom_bar` does not support plotting with negative heights. Instead, bars with negative heights are rendered as having zero height.

### Example:
```nim
import ggplotnim

let trials = @["A", "B", "C", "D", "E"]
let values = @[1.0, 0.5, 0, -0.5, -1.0]

var df = newDataFrame()
df["Trial"] = toColumn(trials)
df["Value"] = toColumn(values)

ggplot(df, aes(x="Trial", y="Value")) + 
  geom_bar(stat="identity", position="identity") + 
  ggsave("demo.png")
```
An equivalent with R/ggplot would produce produce a bar plot with five bars, the first two extending above the x axis, the third sitting on it, and the last two below. However, this snippet instead produces this:

![Not right](https://liamsc.com/misc/demo.png)

**Comment by Vindaar** [comment]
Fixed by #65. I turned your example into a recipe here:

https://github.com/Vindaar/ggplotnim/blob/allowNegBars/recipes.org#bar-plot-with-negative-values

**Comment by Vindaar** [comment]
#65 is merged, closing this.
